cs464-group-project/src/pages/Home.js

```diff
@@ -1,27 +1,171 @@
+          );
+
+          return {
+            id: match.idEvent,
+            homeId: match.idHomeTeam,
+            awayId: match.idAwayTeam,
+            homeName: match.strHomeTeam,
+            awayName: match.strAwayTeam,
+            homeScore: match.intHomeScore,
+            awayScore: match.intAwayScore,
+            eventDate: match.dateEvent,
+            homeLogo: homeTeam ? homeTeam.logo : null,
+            awayLogo: awayTeam ? awayTeam.logo : null,
+          };
+        });
 
+        const groupedMatches = tempPastMatches.reduce((acc, match) => {
+          const date = match.eventDate;
+
+          //create a property for each date storing in an array of matches
+          acc[date] = acc[date] || [];
+          acc[date].push(match);
+          return acc;
+        }, {});
+        setPastMatches(groupedMatches);
+      } catch (err) {
+        console.error('Error getting League information', err);
+      }
+    };
     fetchData();
   }, []);
+
+  //getting live matches
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const data = await getPLLiveScores();
+        if (data.events === null) {
+          return;
+        }
+        const plData = data.events.filter((match) => match.idLeague === '4328');
+        const liveMatches = plData.map((match) => ({
+          homeId: match.idHomeTeam,
+          awayId: match.idAwayTeam,
+          homeTeam: match.strHomeTeam,
+          awayTeam: match.strAwayTeam,
+          homeScore: match.intHomeScore,
+          awayScore: match.intAwayScore,
+          clock: match.strProgress,
+          homeLogo: match.strHomeTeamBadge,
+          awayLogo: match.strAwayTeamBadge,
+        }));
+        setLiveTeams(liveMatches);
+      } catch (err) {
+        console.error('Error getting League information', err);
+      }
+    };
+    fetchData();
+
+    const intervalId = setInterval(() => {
+      fetchData();
+    }, 60000);
+
+    return () => clearInterval(intervalId);
+  }, []);
+
   return (
-    <h1>
-      <img
-        className='league-logo'
-        src='/Images/premierleague_logo.png'
-        alt='Premier League Logo'
-      />
-      Welcome to the Premier League
-    </h1>
+    <>
+      <div className='home-container'>
+        <div className='home-charts'>
+          <Table ranking={ranking} />
+          <div className='gamematches-container'>
+            <div className='live-game'>
+              {liveTeams.length > 0 ? (
+                //if there is live game
+                <>
+                  <h2>Live Matches</h2>
+                  {liveTeams.map((liveMatch) => (
+                    <LiveMatch key={liveMatch.homeId} match={liveMatch} />
+                  ))}
+                </>
+              ) : (
+                <div className='no-live'>
+                  <h2>No Live Games Currently</h2>
+                  <p>Come Back in the Weekends!</p>
+                </div>
+              )}
+            </div>
```

Comment:
I would move the header outside of the gamematches-container so that it lines up the same as the past matches when they are sided to side.